packages/web-components/fast-ssr/src/element-renderer/elemenent-renderer.spec.ts

```diff
@@ -118,6 +118,47 @@ test.describe("FASTElementRenderer", () => {
                 <bare-element  attr><template shadowroot=\"open\"></template></bare-element>
             `);
         });
+        test("should render an attribute with a boolean value for non-aria attributes", () => {
```

Comment:
can you adjust these tests (or maybe just their descriptions?) so that they don't pertain specifically to aria attributes? Since the special casing of aria was removed, I don't want folks reading these tests to get the impression that we are special casing aria.

Reply:
fixed